librz/core/cbin.c

```diff
@@ -4212,16 +4211,16 @@ static void bin_elf_versioninfo_versym(RzCore *r, PJ *pj, int mode) {
 }
 
 static void bin_elf_versioninfo_verneed(RzCore *r, PJ *pj, int mode) {
```

Comment:
Can you also convert the mode into the RzOutputMode enum?

Reply:
no please, this PR has a very different purpose and it's already quite clean as is